diff --git a/autocomplete_deluxe/src/Plugin/Field/FieldWidget/AutocompleteDeluxeWidget.php b/autocomplete_deluxe/src/Plugin/Field/FieldWidget/AutocompleteDeluxeWidget.php
index 3227825..7939d75 100644
--- a/autocomplete_deluxe/src/Plugin/Field/FieldWidget/AutocompleteDeluxeWidget.php
+++ b/autocomplete_deluxe/src/Plugin/Field/FieldWidget/AutocompleteDeluxeWidget.php
@@ -161,10 +161,10 @@ class AutocompleteDeluxeWidget extends WidgetBase implements ContainerFactoryPlu
 
     $new_terms = isset($settings['new_terms']) ? $settings['new_terms'] : FALSE;
     $allow_message = isset($settings['not_found_message_allow']) ? $settings['not_found_message_allow'] : FALSE;
-    $not_found_message = isset($element['not_found_message']) ? $element['not_found_message'] : "The term '@term' will be added.";
+    $not_found_message = isset($settings['not_found_message']) ? $this->t($settings['not_found_message']) : $this->t("The term '@term' will be added.");
     if (!$new_terms) {
       if ($allow_message) {
-        $not_found_message = "Cannot add '@term' because 'Allow new terms' is disabled on the widget settings.";
+        $not_found_message = isset($settings['not_found_message']) ? $this->t($settings['not_found_message']) : $this->t("Cannot add '@term' because 'Allow new terms' is disabled on the widget settings.");
       }
       else {
         $not_found_message = "";
